### ISSUE TITLE
Startup error

### ISSUE THREAD

**Issue by emhouston** [ATGH15102AFMLD/Linkbar#6]
Running latest version 1.6.8
On startup 
"GDI out of memory"

but application seems to run ok.

**Comment by ATGH15102AFMLD** [comment]
Please write additional info.
1)Linkbar bit depth - Linkbar.exe or Linkbar64.exe
2) Linkbar Settings -> About ->System Info. Use context menu for copy system info

**Comment by dkzeanah** [comment]
Im having the same issue.  vrs. 1.6.9, linkbar64.exe. basically when I restart computer, and relaunch linkbar, i get a barrage of these message boxes that say "(GDI+ Error) out of memory). the number of message boxes 'barraged' to me is increasing every time i restart computer, however, when I exit all of them, it seems to work fine, with the ex ception of additional, repeat 'linkbars' presend, which I then exit out of.

**Comment by ATGH15102AFMLD** [comment]
Hello,
This executable collects extended error info - [Linkbar_1_6_9_dev_issue_6.zip](https://1drv.ms/u/s!ArRLDIr9g7QghxKovhtiWPV3WMJx?e=JNIjYS) (3MB).
Please run it, get the error, save a log file and send it to me. Log contains your system information. Don't post the log file here, email me linkbar@yandex.ru
Thank you for your attention to the issue.

**Comment by momodgtl** [comment]
Getting the same error. Everything was running ok for a long time, the only change I made is added another monitor.

**Comment by ATGH15102AFMLD** [comment]
Hello,
Please use the version from my previous message and send me the log.

**Comment by momodgtl** [comment]
It's not available.

**Comment by momodgtl** [comment]
Yup, getting the same GDI+ Error out of memory on startup with that version as well

**Comment by ATGH15102AFMLD** [comment]
Sorry for the delay.
This executable collects extended error info - [Linkbar_1_7_0_dev_debug.zip](https://1drv.ms/u/s!ArRLDIr9g7QghzCEgpnQba40c_0D?e=wrkzAX) (4MB)
Please run it, get the error, save a log file and send it to me. Log contains your system information. Don't post the log file here, email me linkbar@yandex.ru
Thank you for your attention to the issue.

**Comment by FoHlined** [comment]
GDI+ Error) out of memory. Windows 11, using LinkBar64 1.6.9. Of course Win 11 is not supported. Is there a workaround or something? This error pop up every day. 

I am not sure, but i think it happens, when my computer shut down the video signal after 15 minutes. sleep mode is NOT activated.

Edit: I tested it, but it has nothing to do with the energy mode: turn off monitor after 15 minutes.

For now i am trying the Linkbar 1.7.0. beta. If you don't hear anything from me, it works :) Sorry, didn't use it earlier.
